packages/shared/types/staking.ts

```
@@ -253,6 +253,105 @@ export type IEarnTokenItem = {
   info: IToken;
 };
 
+export interface IEarnText {
+  text: string;
+  color?: string;
+}
+
+interface IToken {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'IToken'. Consider to delete it or rename it.

'IToken' is defined here:


(lint/suspicious/noRedeclare)